app/src/lib/menu-update.ts

```diff
@@ -248,6 +250,7 @@ function getRepositoryMenuBuilder(state: IAppState): MenuStateBuilder {
       'create-branch',
       !tipStateIsUnknown && !branchIsUnborn
     )
+    menuStateBuilder.setEnabled('compare-to-branch', hasBranchToCompare)
```

Comment:
Like I mentioned in #4726, we should let the user switch to this view. We already have a helpful message here when you don't have branches to compare to:
<img width="400" src="https://user-images.githubusercontent.com/359239/40208537-96ee1a44-5a7d-11e8-9550-d421be6f4c7b.png">

So this is one step better than our other merge dialog. Can we tidy up this check?